models/safe/base/safe_base_singletons.sql

```diff
@@ -6,11 +6,16 @@
         post_hook='{{ expose_spells(\'["base"]\',
                                     "project",
                                     "safe",
-                                    \'["danielpartida"]\') }}'
+                                    \'["danielpartida", "peterrliem"]\') }}'
     )
 }}
 
 
 -- Fetch all known singleton/mastercopy addresses used via factories.
 select distinct singleton as address
-from {{ source('gnosis_safe_base', 'GnosisSafeProxyFactoryv_1_3_0_evt_ProxyCreation') }}
\ No newline at end of file
+from {{ source('gnosis_safe_base', 'GnosisSafeProxyFactoryv_1_3_0_evt_ProxyCreation') }}
+
+union all
```

Comment:
Seems there are dupes between these 2 sources:
https://dune.com/queries/3729099

Reply:
We may solve this problem by using `union` here.
```suggestion
union
```